addons/website_event_track/static/src/js/event_track_reminder.js

```diff
@@ -82,6 +148,20 @@ publicWidget.registry.websiteEventTrackReminder = publicWidget.Widget.extend({
         }
     },
 
+    _fillIcon: function (ev) {
+        var $el = $(ev.target);
+        if ($el.attr('title') == _t('Set Favorite')){
+            $el.removeClass('fa-bell-o').addClass('fa-bell');
+        }
+    },
+
+    _emptyIcon: function (ev){
+        var $el = $(ev.target);
+        if ($el.attr('title') == _t('Set Favorite')){
+            $el.removeClass('fa-bell').addClass('fa-bell-o');
+        }
+    },
+
```

Comment:
could probably be a single function, something like this (also not sure we really need to differentiate based on the title):
```suggestion
    _onMouseEventUpdateIcon: function (ev) {
        ev.target.classList.toggle('fa-bell-o');
        ev.target.classList.toggle('fa-bell');
    },
```

Reply:
I tried the way you suggested it and other ways to simplify the code, but a js difficulty appeared. When a user click on the bell to add/remove this one of his favorites, the class is change and when he moves out his mouse the class is change once again. So when a bell is filled and the user clicks on it, it becomes empty and when he moves out his mouse it becomes filled, but it should stay empty. So I precised which class must be added according the event and thus I couldn't use the toggle method.         

According to me, if an hover event existed this difficulty shouldn't appear but I didn't find out something similar in Odoo. 

Also, the hover effect is applied only when the bell is empty to invite users to add tracks to favorites. I didn't make it for the opposite case, to not encourage them to remove tracks from their favorites.